gensrc/thrift/HeartbeatService.thrift

```diff
@@ -38,6 +40,7 @@ struct TBackendInfo {
     4: optional Types.TPort brpc_port
     5: optional string version
     6: optional i32 num_hardware_cores
+    7: optional list<WorkGroup.TWorkGroup> active_workgroups
```

Comment:
report work group id is enough？

Reply:
not enough

Reply:
create stmt and alter stmt generate changes to WorkGroup, it should be transfered to BE

Reply:
why create  and alter workgroup need to by heartbeat. which should be a `AgentTask`

Reply:
ok